.github/workflows/release.yml

```diff
@@ -12,7 +12,7 @@ jobs:
     steps:
       - uses: actions/checkout@v2
       - name: Ballerina Build
-        uses: ballerina-platform/ballerina-action/@swan-lake-connector-release
+        uses: ballerina-platform/ballerina-action/@master
         with:
           args:
             build -c --skip-tests
```

Comment:
Need to run tests when releasing? it's ok to skip for specific reason? but the ballerina issue is fixed now

Reply:
Resolved in 57afa94. Thank you.